exercises/make_it_modular/problem.md

```diff
@@ -2,9 +2,9 @@ This problem is the same as the previous but introduces the concept of **modules
 
 Create a program that prints a list of files in a given directory, filtered by the extension of the files. The first argument is the directory name and the second argument is the extension filter. Print the list of files (one file per line) to the console. You **must** use asynchronous I/O.
 
-You must write a *module* file to do most of the work. The module must *export* a single function that takes **three** arguments: the directory name, the filename extension string and a callback function, in that order. The filename extension argument must be the same as what was passed to your program. Don't turn it into a RegExp or prefix with "." or do anything except pass it to your module where you can do what you need to make your filter work.
+You must write a *module* file to do most of the work. The module must *export* a single function that takes **three** arguments: the directory name, the filename extension string and your callback function, in that order. Don't alter the filename extension string before passing it to your module file by turning it into a RegExp, prefixing it with ".", or anything else. Simply pass it to your module where you can then do what you need to make your filter work.
```

Comment:
This line is still a bit complex, let's try to make it little bit more simple ?

Reply:
```
You must write a *module* file to do most of the work. The module must *export* a single function that takes **three** arguments: the directory name, the filename extension string and your callback function, in that order. Don't alter the filename extension string in any way before passing it to your module.
```
More simplified?

Reply:
This sounds good :)